flocker_bb/builders/flocker_vagrant.py

```diff
@@ -357,6 +357,17 @@ def test_installed_package(box):
         tests='flocker',
         workdir='test',
     ))
+    # TODO: FLOC-253
+    # Quick ugly hack to get the test.log output out of the vagrant ssh
```

Comment:
If you have a good plan for how to do this in a nicer way, add a new JIRA issue and link to it.

If you don't have a good plan, then this isn't a quick ugly hack. It's the solution.